pyccel/ast/numpyext.py

```diff
@@ -115,6 +117,26 @@
     'NumpyShape',
 )
 
+class NumpyNDArrayType(DataType, metaclass=Singleton):
```

Comment:
PyccelBot says that

>`numpyext.NumpyNDArrayType` is missing from `__all__`

Is this intentional?

Reply:
No fixed [ea99fbf](https://github.com/pyccel/pyccel/pull/1590/commits/ea99fbf49e06896774f7a901a333666a1eacd759)